driver/src/lib.rs

```diff
@@ -5,6 +5,8 @@
 //   we successfully link.
 extern crate ethereum_tx_sign;
 extern crate web3;
+#[macro_use]
+extern crate lazy_static;
```

Comment:
This is not necessary, in Rust edition 2018 it is generally preferred to locally import macros with a use statement like this:
```
use lazy_static::lazy_static;
```

The crate documentation for `lazy_static` looks to still be in Rust edition 2015 :/